opentech/apply/projects/forms.py

```diff
@@ -107,6 +108,26 @@ def __init__(self, user=None, *args, **kwargs):
         super().__init__(*args, initial=initial, **kwargs)
 
 
+class EditPaymentRequestForm(forms.ModelForm):
+    receipt_list = forms.MultipleChoiceField(widget=forms.CheckboxSelectMultiple(attrs={'checked': 'checked'}))
+
+    name = 'edit_payment_request_form'
+
+    class Meta:
+        fields = ['invoice', 'value', 'date_from', 'date_to', 'receipt_list', 'comment']
+        model = PaymentRequest
+
+    def __init__(self, user=None, instance=None, *args, **kwargs):
+        super().__init__(*args, instance=instance, **kwargs)
+
+        self.instance = instance
+
+        self.fields['receipt_list'].choices = [
+            (r.pk, os.path.basename(r.file.url))
```

Comment:
Should be working with the `file.name` because storage messes with the basename in the url?

Reply:
I'm getting the same output for `file.name` and `file.url` after passing through `os.path.basename`.  I looked into `SubmissionStreamFileField` which has a `filename` but that's the same output for me locally too so doesn't gain us anything.